extension/src/plots/model/collect.test.ts

```diff
@@ -215,50 +172,6 @@ describe('collectData', () => {
   })
 })
 
-describe('collectCustomCheckpointPlotsData', () => {
-  it('should return the expected data from the test fixture', () => {
-    const data = collectCustomCheckpointPlots(expShowFixture)
-
-    expect(data).toStrictEqual(checkpointPlotsFixture)
-  })
-
-  it('should provide a continuous series for a modified experiment', () => {
```

Comment:
[F] This logic has been removed on purpose. Change is being discussed in `[WIP] exp: refactor show behavior` PR. Link is further down.